modules/ROOT/pages/import-api-specification-local-file.adoc

```diff
@@ -0,0 +1,37 @@
+= Importing an API Specification from a Local File
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio allows you to import an API specification from your local machine and import it either into a new Mule Project, or an existing one.
+
+This task only works for RAML types of API specifications.
+
+. In the top task bar, select *File*, then select *New*, and then *Mule Project*.
+. Type a name for your project.
+. Select a Mule runtime version.
+. In the *API Specification* section, select the *From Design Center or Local Files* tab.
+. In the *Location* field, type the location of your local API, or click the *...* button and select *Browse Files...*.
+You can import a RAML file, or an entire zip file bundle if your API has schemas and examples.
+. Click *Finish*.
+
+Studio scaffolds your API specification. If you choose not to create flows out of your API specification, uncheck the *Scaffold Flows From These API Specifications* option.
+
+You can import more than one API. You can select the API you want to scaffold flows and check the *Scaffold selected API specifications* checkbox so that Studio generates the flow based on the imported API.
+
+It is not possible to add a local API specification to an existing Mule project.
```

Comment:
This sounds like something that belongs in the intro text, so that readers proceed no further if they want to do this....

Also...is this here only until the following task is validated?